### PR TITLE
CD: Update Content Blockers rules

### PR DESCRIPTION
**Pull request by chrmod**
This CD script will update Content Blocker rules to the latest version. That way we will always test latest rules in Nightly builds.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -147,6 +147,8 @@ node('gideon') {
 
                                 export MATCH_KEYCHAIN_NAME=ios-build.keychain
 
+				npm run update-content-blocker
```
Comment (remusao) [inline Jenkinsfile]: Weird indentation!

File: scripts/update-content-blocker.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  const adCosmeticRulesUrl = adList.safari.cosmetic;
+  const adCosmeticRules = await (await fetch(adCosmeticRulesUrl)).text();
+  fs.writeFileSync('content-blocker-lib-ios/Lists/safari-ads-cosmetic.json', adCosmeticRules);
+
+  const trackingList = await (await fetch("https://cdn.cliqz.com/adblocker/configs/safari-tracking/allowed-lists.json")).json();
+
+  // Tracking Network rules
+  const trackingNetworkRulesUrl = trackingList.safari.network;
+  const trackingNetworkRules = await (await fetch(trackingNetworkRulesUrl)).text();
+  fs.writeFileSync('content-blocker-lib-ios/Lists/safari-tracking-network.json', trackingNetworkRules);
```
Comment (remusao) [inline scripts/update-content-blocker.js]: Maybe it's overkill but you might be able to abstract this a bit?

```js
async function getRules(url, path) {
  const trackingNetworkRules = await (await fetch(trackingNetworkRulesUrl)).text();
  if (trackingNetworkRules !== '[]') {
    fs.promises.writeFile(path, trackingNetworkRules);
  }
}

async function getLists(config) {
  const trackingList = await (await fetch(`https://cdn.cliqz.com/adblocker/configs/${config}/allowed-lists.json"`)).json();
  await Promise.all([
    getRules(trackingList.safari.network, `content-blocker-lib-ios/Lists/${config}-network.json`),
    getRules(trackingList.safari.cosmetic, `content-blocker-lib-ios/Lists/${config}-cosmetic.json`),
  ]);
}

(async () => {
  await Promise.all([
    getLists('safari-ads'),
    getLists('safari-tracking'),
  ]);
})();
```

Comment (chrmod) [reply scripts/update-content-blocker.js]: Lets keep is simple for now :-)

Comment (remusao) [reply scripts/update-content-blocker.js]: @chrmod I am happy to let you pick the definition of simplicity which suits you best!

**Pull request merged**